### PR TITLE
[Fix more model errors] Support analysis in delete fast. 

### PR DESCRIPTION
**Pull request by 2742195759**

### CODE REVIEW

File: sot/opcode_translator/executor/opcode_executor.py [hunk 1]
```diff
@@ -1588,7 +1604,17 @@ def _break_graph_in_call(
             for name in resume_input_name
             if self.get_var(name) not in ret_vars
         ]
-        self._graph.start_compile(*ret_vars)
+
+        # Collect all the to store variables.
```
Comment (SigureMo) [inline sot/opcode_translator/executor/opcode_executor.py:1608]: 这里是 all the 啥 :joy:

**Review by SigureMo — APPROVED** [review]
LGTM~~~

File: sot/opcode_translator/executor/opcode_executor.py (hunk 1 above)
Comment (2742195759) [reply sot/opcode_translator/executor/opcode_executor.py:1608]: to store variables ...

Comment (SigureMo) [reply sot/opcode_translator/executor/opcode_executor.py:1608]: 喔喔 蠢了

**Review by feifei-111 — APPROVED** [review]
LGTM

**Pull request merged**